openshift_cli_installer/utils/clusters.py

```diff
@@ -162,3 +165,40 @@ def add_s3_bucket_data(clusters, s3_bucket_name, s3_bucket_path=None):
         )
 
     return clusters
+
+
+def collect_must_gather(must_gather_output_dir, cluster_data, cluster_object=None):
+    name = cluster_data["name"]
```

Comment:
everything needed to go under the `try`, we don't want to fail here never (we cal destroy after)